4788/src/main/cpp/Strategy/ShooterStrategy.cpp

```diff
@@ -0,0 +1,26 @@
+#include "Strategy/ShooterStrategy.h"
+#include <iostream>
+
+ShooterManualStrategy::ShooterManualStrategy(std::string name, Shooter &shooter, Controllers &contGroup) : Strategy(name), _shooter(shooter), _contGroup(contGroup) {
+  SetCanBeInterrupted(true);
+  SetCanBeReused(true);
+  Requires(&shooter);
+
+}
+
+void ShooterManualStrategy::OnUpdate(double dt) {
+  double manualFlyWheelPower = fabs(_contGroup.Get(ControlMap::manualFlyWheel)) > fabs(ControlMap::xboxDeadzone) ? _contGroup.Get(ControlMap::manualFlyWheel) : 0;
+
+  if (_contGroup.Get(ControlMap::innerCircleShoot)) {
+    _shooter.setAuto(200, dt);
```

Comment:
Put these constants in `RobotMap` or `ControlMap` in order to make them easier to find and reconfigurable.